public/app/features/alerting/unified/components/rule-editor/DashboardPicker.tsx

```diff
@@ -195,12 +210,16 @@ export const DashboardPicker = ({ dashboardUid, panelId, isOpen, onChange, onDis
         </div>
 
         <div className={styles.column}>
-          {!dashboardUid && !isDashboardFetching && <div>Select a dashboard to get a list of available panels</div>}
+          {!selectedDashboardUid && !isDashboardFetching && (
+            <div className={styles.selectDashboardPlaceholder}>
+              <div>Select a dashboard to get a list of available panels</div>
+            </div>
+          )}
           {isDashboardFetching && (
             <LoadingPlaceholder text="Loading dashboard..." className={styles.loadingPlaceholder} />
           )}
 
-          {!isDashboardFetching && (
+          {selectedDashboardUid && !isDashboardFetching && (
```

Comment:
Why we need to add `selectedDashboardUid` to this expression ?

Reply:
To prevent displaying an empty grid and be able to vertically center the `Select a dashboard...` text